src/lib.rs

```diff
@@ -353,9 +353,19 @@ impl<'a> Extract<'a> {
                 let mut archive = zip::ZipArchive::new(source)?;
                 for i in 0..archive.len() {
                     let mut file = archive.by_index(i)?;
-                    let path = into_dir.join(file.name());
-                    let mut output = fs::File::create(path)?;
-                    io::copy(&mut file, &mut output)?;
+                    let outpath = into_dir.join(file.sanitized_name());
+
+                    if (&*file.name()).ends_with('/') {
+                        fs::create_dir_all(&outpath).unwrap();
```

Comment:
can you use the `?` operator instead of `unwrap`s

Reply:
Fixed